pkg/app/web/src/components/application-form/index.tsx

```diff
@@ -83,6 +86,66 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+type TabPanelProps = {
+  children: any
+  value: any
+  index: any
+}
```

Comment:
should avoid using `any` type, the check `props.value !== props.index` at line 100 is unsafe due to comparing between two values of type any, if it's a number, use type `number` instead.

Reply:
For the `children` field, is type `HTMLElement` good enough?

Reply:
Right. Let me look into it.